### ISSUE TITLE
Allow passing Icon into Write-Log for Slack Target

### ISSUE THREAD

**Issue by MCrank** [EsOsO/Logging#53]
* As a user I would like to be able to pass in a custom icon into the Slack Target to override the existing ones.
* Today there is no way to send a message to one target vs another that I can see.  I have created a Specific Logging Level that handles the SLACK messages and that is working.  The problem is the Slack target module supplies the 🔥 ⚠️ ❗️ icons based on logging value so all of my slack messages are coming in with FIRE icon since I made the logging level 50
* Would be nice to allow passing in an icon from my slack org for each message that I am sending

* I tried setting three new Logging-Levels (25 -SLACKINFO, 35 -SLACKWARN, 50- SLACKERROR) but since the -Level param for the Slack target is a String and that is how I am targeting the Slack logging target no messages were being sent.  I could be doing this wrong as well.

**Comment by EsOsO** [comment]
Hi @MCrank, thanks for the suggestion, I'm against putting a target specific parameter to `Write-Log`. I'm planning to put an hashtable as an added parameter in the Slack target configuration:
```powershell
Add-LoggingTarget -Name Slack -Configuration @{Level = 'DEBUG'; Icons = @{
   'ERROR' = ':error_emoji:'
   'WARNING' = ':warning_emoji'
   'INFO' = ':info_emoji:'
   'DEBUG' = ':debug_emoji:'
}}
```
You'll be able to map a Level to an emoji and use only the icons relevant to you.
If sounds good to you I'll publish the patch.

**Comment by MCrank** [comment]
@EsOsO That looks like it may work.  I'll try to grab this once it gets merged and give it a run.  Thanks.

**Comment by EsOsO** [comment]
Closed in 4.1.1
